lib/model/game_state.dart

```diff
@@ -80,6 +81,12 @@ class GameState with ChangeNotifier {
     notifyListeners();
   }
 
+  List<Card> getActiveQuestCards() {
+    final discardQuestCards = discard.reversed.where((card) => card.type == CardType.QUEST).toList();
+    final deckQuestCards = deck.reversed.where((card) => card.type == CardType.QUEST).toList();
+    return discardQuestCards + deckQuestCards;
```

Comment:
We should probably also include the current card here, otherwise it doesn't show in active quests. If we do the combining up front we can simplify a bit:

```suggestion
    final allCards = discard + deck + [currentCard];
    return allCards.where((card) => card.type == CardType.QUEST).toList();
```

Reply:
Also, just out of curiosity really, what's the thinking behind `reversed`?

We use it for the discard view because it makes semantic sense to see the most recent card "on top" of it. But I think for the active quests view, where there's only ever going to be (at most) 2 cards showing atm, it doesn't matter what order we show them in?

Reply:
The thinking behind reversed was basically that the discard view had it, but what you've said makes more sense. I have removed it on the basis that less complexity is good if we don't need it. I've also changed the test to be agnostic about the ordering. I'll address the current card bit next.

Reply:
I've pushed a change to include the current card in active quest cards if it's a quest card. Now that I think about it, in the case that the current card is a quest card, would it make sense to expect that one to be displayed first?

Reply:
Considering possible edge cases - and this might reveal a gap in my understanding of the game - is it possible for there to be no active quest cards?

Reply:
Yeah, it is - I tested that case when playing around and nothing breaks (but you do get taken to a slightly odd-looking empty screen).

Probably not worth worrying about at this stage, though - because if you get into that state then the game is unwinnable anyway. The expectation is that the gameplay will change quite a bit over time - there'll be balance changes, new mechanics, new quest lines and so on.